spec/requests/case_court_reports_spec.rb

```diff
@@ -125,6 +124,17 @@
 
           expect(get_docx_subfile_contents(response.body, "word/header3.xml")).to include("YOUR CASA ORG’S NUMBER")
         end
+
+        # TODO admin and supervisor tests instruction in wrong order
```

Comment:
yes tests please

Reply:
out of the scope of this issue and the test is wrong because it generates the docx download link as a volunteer

Reply:
interesting... can you make a chore about it plz